### PR TITLE
BUG Fix form submission

### PR DESCRIPTION
**Pull request by tractorcow**

### CODE REVIEW

**Comment by flashbackzoo** [comment]
Couple of issues..

JS error after submission:

userform-submit-error

The form is only 50% width now:

userforms-css-issue

**Comment by tractorcow** [comment]
I've added fixes for the JS error as well as some display logic as well as https://github.com/open-sausages/silverstripe-userforms/pull/16

I don't know what to say about width 50%.... I think that's ALWAYS BEEN THERE, not just in this pr. :)

File: javascript/UserForm.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ jQuery(function ($) {
 	 */
 	UserForm.prototype.setCurrentStep = function (step) {
 		// Make sure we're dealing with a form step.
-		if (!step instanceof FormStep) {
+		if (!(step instanceof FormStep)) {
```
Comment (tractorcow) [inline javascript/UserForm.js]: LMFTFY

Comment (flashbackzoo) [inline javascript/UserForm.js]: Ah! Well spotted :smile:

**Pull request merged**